### ISSUE TITLE
2021-03

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#16]

**Comment by qiaojy19** [comment]
# Mar - 2021

## 03.29 - 04.02
| Title                                                                                                                                                           |
| --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [1-Bit Compressive Sensing for Efficient Federated Learning Over the Air](https://arxiv.org/abs/2103.16055)                                                     |
| [A Federated Learning Framework for Non-Intrusive Load Monitoring](https://arxiv.org/abs/2104.01618)                                                            |
| [A Federated Learning Framework for Smart Grids: Securing Power Traces in Collaborative Learning](https://arxiv.org/abs/2103.11870)                             |
| [An Empirical Evaluation of Cost-based Federated SPARQL Query Processing Engines](https://arxiv.org/abs/2104.00984)                                             |
| [An On-Device Federated Learning Approach for Cooperative Anomaly Detection](https://arxiv.org/abs/2002.12301)                                                  |
| [Auto-weighted Robust Federated Learning with Corrupted Data Sources](https://arxiv.org/abs/2101.05880)                                                         |
| [Certifiably-Robust Federated Adversarial Learning via Randomized Smoothing](https://arxiv.org/abs/2103.16031)                                                  |
| [Decentralized and Model-Free Federated Learning: Consensus-Based Distillation in Function Space](https://arxiv.org/abs/2104.00352)                             |
| [Delay Analysis of Wireless Federated Learning Based on Saddle Point Approximation and Large Deviation Theory](https://arxiv.org/abs/2103.16994)                |
| [Demonstrating Analog Inference on the BrainScaleS-2 Mobile System](https://arxiv.org/abs/2103.15960)                                                           |
| [Fast-adapting and Privacy-preserving Federated Recommender System](https://arxiv.org/abs/2104.00919)                                                           |
| [Federated Few-Shot Learning with Adversarial Learning](https://arxiv.org/abs/2104.00365)                                                                       |
| [Federated Generalized Bayesian Learning via Distributed Stein Variational Gradient Descent](https://arxiv.org/abs/2009.06419)                                  |
| [Federated Learning with Taskonomy for Non-IID Data](https://arxiv.org/abs/2103.15947)                                                                          |
| [Federated Learning: A Signal Processing Perspective](https://arxiv.org/abs/2103.17150)                                                                         |
| [Federated Semi-Supervised Learning with Inter-Client Consistency & Disjoint Learning](https://arxiv.org/abs/2006.12097)                                        |
| [FedMood:Federated Learning on Mobile Health Data for Mood Detection](https://arxiv.org/abs/2102.09342)                                                         |
| [Intelligent Radio Signal Processing: A Survey](https://arxiv.org/abs/2008.08264)                                                                               |
| [Learning Rate Optimization for Federated Learning Exploiting Over-the-air Computation](https://arxiv.org/abs/2102.02946)                                       |
| [Model-Contrastive Federated Learning](https://arxiv.org/abs/2103.16257)                                                                                        |
| [On the Convergence Time of Federated Learning Over Wireless Networks Under Imperfect CSI](https://arxiv.org/abs/2104.00331)                                    |
| [Privacy and Trust Redefined in Federated Machine Learning](https://arxiv.org/abs/2103.15753)                                                                   |
| [PyVertical: A Vertical Federated Learning Framework for Multi-headed SplitNN](https://arxiv.org/abs/2104.00489)                                                |
| [SecureBoost: A Lossless Federated Learning Framework](https://arxiv.org/abs/1901.08755)                                                                        |
| [Semi-supervised Federated Learning for Activity Recognition](https://arxiv.org/abs/2011.00851)                                                                 |
| [Towards open and expandable cognitive AI architectures for large-scale multi-agent human-robot collaborative learning](https://arxiv.org/abs/2012.08174)       |
| [User profile-driven large-scale multi-agent learning from demonstration in federated human-robot collaborative environments](https://arxiv.org/abs/2103.16434) |

## 03.22 - 03.26
| Title                                                                                                                                                 |
| ----------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Convergence Time Optimization for Federated Learning over Wireless Networks](https://arxiv.org/abs/2001.07845)                                       |
| [Ensemble Distillation for Robust Model Fusion in Federated Learning](https://arxiv.org/abs/2006.07242)                                               |
| [Quality Inference in Federated Learning with Secure Aggregation](https://arxiv.org/abs/2007.06236)                                                   |
| [Flower: A Friendly Federated Learning Research Framework](https://arxiv.org/abs/2007.14390)                                                          |
| [X-Secure T-Private Federated Submodel Learning with Elastic Dropout Resilience](https://arxiv.org/abs/2010.01059)                                    |
| [Minimal conditions analysis of gradient-based reconstruction in Federated Learning](https://arxiv.org/abs/2010.15718)                                |
| [FedRec: Federated Learning of Universal Receivers over Fading Channels](https://arxiv.org/abs/2011.07271)                                            |
| [Personalized Federated Learning with First Order Model Optimization](https://arxiv.org/abs/2012.08565)                                               |
| [Differentially Private Federated Learning for Cancer Prediction](https://arxiv.org/abs/2101.02997)                                                   |
| [Federated Reconstruction: Partially Local Federated Learning](https://arxiv.org/abs/2102.03448)                                                      |
| [A Federated Learning Framework in Smart Grid: Securing Power Traces in Collaborative Learning](https://arxiv.org/abs/2103.11870)                     |
| [Real-time End-to-End Federated Learning: An Automotive Case Study](https://arxiv.org/abs/2103.11879)                                                 |
| [Federated Quantum Machine Learning](https://arxiv.org/abs/2103.12010)                                                                                |
| [The Gradient Convergence Bound of Federated Multi-Agent Reinforcement Learning with Efficient Communication](https://arxiv.org/abs/2103.13026)       |
| [Opportunistic Federated Learning: An Exploration of Egocentric Collaboration for Pervasive Computing Applications](https://arxiv.org/abs/2103.13266) |
| [Benchmarking Modern Named Entity Recognition Techniques for Free-text Health Record De-identification](https://arxiv.org/abs/2103.13546)             |
| [FedGP: Correlation-Based Active Client Selection for Heterogeneous Federated Learning](https://arxiv.org/abs/2103.13822)                             |
| [Hierarchical Quantized Federated Learning: Convergence Analysis and System Design](https://arxiv.org/abs/2103.14272)                                 |
| [Prior-Free Auctions for the Demand Side of Federated Learning](https://arxiv.org/abs/2103.14375)                                                     |

## 03.15 - 03.19
| Title                                                                                                                                               |
| --------------------------------------------------------------------------------------------------------------------------------------------------- |
| [FLAME: Differentially Private Federated Learning in the Shuffle Model](https://arxiv.org/abs/2009.08063)                                           |
| [Sample-based Federated Learning via Mini-batch SSCA](https://arxiv.org/abs/2103.09506)                                                             |
| [CaPC Learning: Confidential and Private Collaborative Learning](https://arxiv.org/abs/2102.05188)                                                  |
| [UAV Communications for Sustainable Federated Learning](https://arxiv.org/abs/2103.11073)                                                           |
| [A Framework for Energy and Carbon Footprint Analysis of Distributed and Federated Edge Learning](https://arxiv.org/abs/2103.10346)                 |
| [An Experiment Study on Federated LearningTestbed](https://arxiv.org/abs/2103.10579)                                                                |
| [Bias-Free FedGAN](https://arxiv.org/abs/2103.09876)                                                                                                |
| [Blind Federated Learning at the Wireless Edge with Low-Resolution ADC and DAC](https://arxiv.org/abs/2010.00350)                                   |
| [Demystifying the Effects of Non-Independence in Federated Learning](https://arxiv.org/abs/2103.11226)                                              |
| [Privacy Preservation in Federated Learning: An insightful survey from the GDPR Perspective](https://arxiv.org/abs/2011.05411)                      |
| [QoS-Constrained Federated Learning Empowered by Intelligent Reflecting Surface](https://arxiv.org/abs/2103.11551)                                  |
| [Reconfigurable Intelligent Surface Enabled Federated Learning: A Unified Communication-Learning Design Approach](https://arxiv.org/abs/2011.10282) |
| [R-GAP: Recursive Gradient Attack on Privacy](https://arxiv.org/abs/2010.07733)                                                                     |
| [Server Averaging for Federated Learning](https://arxiv.org/abs/2103.11619)                                                                         |
| [TAG: Transformer Attack from Gradient](https://arxiv.org/abs/2103.06819)                                                                           |
| [Toward Robustness and Privacy in Federated Learning: Experimenting with Local and Central Differential Privacy](https://arxiv.org/abs/2009.03561)  |
| [Two Timescale Hybrid Federated Learning with Cooperative D2D Local Model Aggregations](https://arxiv.org/abs/2103.10481)                           |
| [GS-WGAN: A Gradient-Sanitized Approach for Learning Differentially Private Generators](https://arxiv.org/abs/2006.08265)                           |
## 03.08 - 03.12

| Title                                                                                                                                                                               |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Convergence and Accuracy Trade-Offs inFederatedLearningand Meta-Learning](https://arxiv.org/abs/2103.05032)                                                                        |
| [FedDG: Federated Domain Generalization on Medical Image Segmentation via Episodic Learning in Continuous Frequency Space](https://arxiv.org/abs/2103.06030)                        |
| [Multi-institutional Collaborations for Improving DeepLearning-based Magnetic Resonance Image Reconstruction UsingFederatedLearning](https://arxiv.org/abs/2103.02148)              |
| [A Better Alternative to Error Feedback for Communication-Efficient Distributed Learning](https://arxiv.org/abs/2006.11077)                                                         |
| [PFA: Privacy-preservingFederatedAdaptation for Effective Model Personalization](https://arxiv.org/abs/2103.01548)                                                                  |
| [A Tree-based Federated Learning Approach for Personalized Treatment Effect Estimation from Heterogeneous Data Sources](https://arxiv.org/abs/2103.06261)                           |
| [Advances and Open Problems inFederatedLearning](https://arxiv.org/abs/1912.04977)                                                                                                  |
| [Applications of Federated Learning in Smart Cities: Recent Advances, Taxonomy, and Open Challenges](https://arxiv.org/abs/2102.01375)                                              |
| [Auction Based Clustered Federated Learning in Mobile Edge Computing System](https://arxiv.org/abs/2103.07150)                                                                      |
| [Bidirectional compression in heterogeneous settings for distributed orfederatedlearningwith partial participation: tight convergence guarantees](https://arxiv.org/abs/2006.14591) |
| [Characterizing Impacts of Heterogeneity in Federated Learning upon Large-Scale Smartphone Data](https://arxiv.org/abs/2006.06983)                                                  |
| [DecentralisedLearningfrom Independent Multi-Domain Labels for Person Re-Identification](https://arxiv.org/abs/2006.04150)                                                          |
| [Deploying Federated Learning in Large-Scale Cellular Networks: Spatial Convergence Analysis](https://arxiv.org/abs/2103.06056)                                                     |
| [Differentially Private Learning with Adaptive Clipping](https://arxiv.org/abs/1905.03871)                                                                                          |
| [DynamicFederatedLearning-Based Economic Framework for Internet-of-Vehicles](https://arxiv.org/abs/2101.00191)                                                                      |
| [Federated Functional Gradient Boosting](https://arxiv.org/abs/2103.06972)                                                                                                          |
| [FedMood:FederatedLearningon Mobile Health Data for Mood Detection](https://arxiv.org/abs/2102.09342)                                                                               |
| [FLBench: A Benchmark Suite for Federated Learning](https://arxiv.org/abs/2008.07257)                                                                                               |
| [Improving Semi-supervisedFederatedLearningby Reducing the Gradient Diversity of Models](https://arxiv.org/abs/2008.11364)                                                          |
| [Multi-Task Federated Reinforcement Learning with Adversaries](https://arxiv.org/abs/2103.06473)                                                                                    |
| [PeerLearningfor Skin Lesion Classification](https://arxiv.org/abs/2103.03703)                                                                                                      |
| [Personalized Federated Learning using Hypernetworks](https://arxiv.org/abs/2103.04628)                                                                                             |
| [Private Cross-Silo Federated Learning for Extracting Vaccine Adverse Event Mentions](https://arxiv.org/abs/2103.07491)                                                             |
| [Provably Efficient Cooperative Multi-Agent ReinforcementLearningwith Function Approximation](https://arxiv.org/abs/2103.04972)                                                     |
| [SCEI: A Smart-Contract Driven Edge Intelligence Framework for IoT Systems](https://arxiv.org/abs/2103.07050)                                                                       |
| [Simeon -- Secure Federated Machine Learning Through Iterative Filtering](https://arxiv.org/abs/2103.07704)                                                                         |
| [TAG: Transformer Attack from Gradient](https://arxiv.org/abs/2103.06819)                                                                                                           |
| [Timely Communication in Federated Learning](https://arxiv.org/abs/2012.15831)                                                                                                      |
## 03.01 - 03.05


| Title                                                                                                                                                                         |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Federated Multi-Armed Bandits](https://arxiv.org/abs/2101.12204)                                                                                                             |
| [Secure Bilevel Asynchronous Vertical Federated Learning with Backward Updating](https://arxiv.org/abs/2103.00958)                                                            |
| [Multi-institutional Collaborations for Improving Deep Learning-based Magnetic Resonance Image Reconstruction Using Federated Learning](https://arxiv.org/abs/2103.02148)     |
| [Linear Regression over Networks with Communication Guarantees](https://arxiv.org/abs/2103.04140)                                                                             |
| [PFA: Privacy-preserving Federated Adaptation for Effective Model Personalization](https://arxiv.org/abs/2103.01548)                                                          |
| [A review of deeplearningin medical imaging: Imaging traits, technology trends, case studies with progress highlights, and future promises](https://arxiv.org/abs/2008.09104) |
| [A Theorem of the Alternative for Personalized Federated Learning](https://arxiv.org/abs/2103.01901)                                                                          |
| [A Unified Theory of Decentralized SGD with Changing Topology and Local Updates](https://arxiv.org/abs/2003.10422)                                                            |
| [Adaptive Transmission Scheduling in Wireless Networks for Asynchronous Federated Learning](https://arxiv.org/abs/2103.01422)                                                 |
| [Adversarial training in communication constrained federated learning](https://arxiv.org/abs/2103.01319)                                                                      |
| [ARIANN: Low-Interaction Privacy-Preserving Deep Learning via Function Secret Sharing](https://arxiv.org/abs/2006.04593)                                                      |
| [Blockchain-Based Federated Learning in Mobile Edge Networks with Application in Internet of Vehicles](https://arxiv.org/abs/2103.01116)                                      |
| [Channel-Driven Monte Carlo Sampling for Bayesian Distributed Learning in Wireless Data Centers](https://arxiv.org/abs/2103.01351)                                            |
| [Differentially Private Learning with Adaptive Clipping](https://arxiv.org/abs/1905.03871)                                                                                    |
| [Distributed Dynamic Map Fusion viaFederatedLearningfor Intelligent Networked Vehicles](https://arxiv.org/abs/2103.03786)                                                     |
| [Edge-assisted Democratized Learning Towards Federated Analytics](https://arxiv.org/abs/2012.00425)                                                                           |
| [Evaluation and Optimization of Distributed Machine Learning Techniques for Internet of Things](https://arxiv.org/abs/2103.02762)                                             |
| [FedDis: DisentangledFederatedLearningfor Unsupervised Brain Pathology Segmentation](https://arxiv.org/abs/2103.03705)                                                        |
| [Federated Learning with Randomized Douglas-Rachford Splitting Methods](https://arxiv.org/abs/2103.03452)                                                                     |
| [FederatedTransferLearning: concept and applications](https://arxiv.org/abs/2010.15561)                                                                                       |
| [FedGNN: Federated Graph Neural Network for Privacy-Preserving Recommendation](https://arxiv.org/abs/2102.04925)                                                              |
| [FedV: Privacy-PreservingFederatedLearningover Vertically Partitioned Data](https://arxiv.org/abs/2103.03918)                                                                 |
| [FjORD: Fair and Accurate Federated Learning under heterogeneous targets with Ordered Dropout](https://arxiv.org/abs/2102.13451)                                              |
| [Helios: Heterogeneity-Aware Federated Learning with Dynamically Balanced Collaboration](https://arxiv.org/abs/1912.01684)                                                    |
| [Learning Rate Optimization for Federated Learning Exploiting Over-the-air Computation](https://arxiv.org/abs/2102.02946)                                                     |
| [Local Averaging Helps: HierarchicalFederatedLearningand Convergence Analysis](https://arxiv.org/abs/2010.12998)                                                              |
| [Moshpit SGD: Communication-Efficient Decentralized Training on Heterogeneous Unreliable Devices](https://arxiv.org/abs/2103.03239)                                           |
| [One for One, or All for All: Equilibria and Optimality of Collaboration in Federated Learning](https://arxiv.org/abs/2103.03228)                                             |
| [Optimization of User Selection and Bandwidth Allocation forFederatedLearningin VLC/RF Systems](https://arxiv.org/abs/2103.03444)                                             |
| [Personalized Federated Learning with Moreau Envelopes](https://arxiv.org/abs/2006.08848)                                                                                     |
| [Privacy Amplification for Federated Learning via User Sampling and Wireless Aggregation](https://arxiv.org/abs/2103.01953)                                                   |
| [Privacy-Preserving Federated Depression Detection from Multi-Source Mobile Health Data](https://arxiv.org/abs/2102.09342)                                                    |
| [R-Learning Based Admission Control for Service Federation in Multi-domain 5G Networks](https://arxiv.org/abs/2103.02964)                                                     |
| [Temporal-Structure-Assisted Gradient Aggregation for Over-the-Air Federated Edge Learning](https://arxiv.org/abs/2103.02270)                                                 |
| [ZeroSARAH: Efficient Nonconvex Finite-Sum Optimization with Zero Full Gradient Computation](https://arxiv.org/abs/2103.01447)                                                |
| [ESMFL: Efficient and Secure Models for Federated Learning](https://arxiv.org/abs/2009.01867)                                                                                 |
